test/python/docker/test_images.py

```diff
@@ -146,6 +146,15 @@ def test_load_image(self):
 
         self.assertEqual(len(self.client.images.list()), 2)
 
+    def test_build_image(self):
+        with self.assertRaises(errors.NotFound):
+            labels = {"apple": "red", "grape": "green"}
+            _ = self.client.images.build(path="build_labels", labels=labels, tag="labels")
+            image = self.client.images.get("labels")
+            self.assertEqual(image.labels["apple"], labels["apple"])
+            self.assertEqual(image.labels["grape"], labels["grape"])
```

Comment:
```suggestion
            self.assertListEqual(image.labels, labels)
```
Provides better diagnostics